### PR TITLE
[DevTestLabs] Adding show, delete & list commands for environment in the lab

### PR DESCRIPTION
**Pull request by vishrutshah**
This is part of feature https://github.com/Azure/azure-cli/issues/2860

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-lab/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+(2017-04-18)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-lab/HISTORY.rst]: This should say "unreleased"

**Comment by tjprescott** [comment]
Also, why not simply include the create and update commands in the same PR?

**Comment by vishrutshah** [comment]
@tjprescott create/update are bit more complex and I am still having conversations on what the experience should look like so I'd like to keep things small and incremental as long as we can.

**Comment by vishrutshah** [comment]
as @tjprescott mentioned I'll wait for the things to be resolved around creation scenario.

**Comment by vishrutshah** [comment]
Merging this PR as I've ready to go PR for the creating environments for lab and hopefully we'd be able to make it in :) 

Thanks for the review, guys!

**Pull request merged**